### DISCUSSION TITLE
Error on private page

### DISCUSSION THREAD

**Discussion by da0142857** [aclap-dev/video-downloadhelper#670] [Help and Q&A]
### Your question

I'm getting this error when downloading from olympus.mygreatlearning.com platform

### Addon & CoApp version and details

{
  "version": "9.0.1.13",
  "target": "mozilla",
  "channel": "stable",
  "page_url": "https://olympus.mygreatlearning.com/courses/99768/pages/optional-content-transfer-learning-architectures-vgg16?module_item_id=5739016",
  "error": {
    "error": "coapp_failure",
    "details": "SideDownload error: : [hls @ 0x519dc80] Error when loading first segment 'https://d3tejxpotd0487.cloudfront.net/HLS/production/output-c0c11434-ee0d-477b-b37e-26139609787f-1080p00000.ts'\n[in#0 @ 0x519db40] Error opening input: Invalid data found when processing input\nError opening input file https://d3tejxpotd0487.cloudfront.net/HLS/production/output-c0c11434-ee0d-477b-b37e-26139609787f-1080p.m3u8.\nError opening input files: Invalid data found when processing input\n",
    "id": "coapp_failure_cdf753bd-7368-4703-b287-ae325c9cb6d2",
    "downloadable_id": "downloadable_1918298622415172"
  }
}

**Comment by GrampaWildWilly** [comment]
I do not work for VDH.  I'm just another user.

I tried the URL in your page_url tag, & I tried the second URL in your error messages.  The first URL just prompted me for a login ID & password.  I suppose that's no surprise.  The second URL gave me a 403 Forbidden error.  I suppose that's not much of a surprise either.

Given that I can't investigate this, & I'm not about to sign up for the site, & I won't allow you to share your login credentials with me, there is only 1 other idea I have.  This sort of error sometimes happens when an antivirus program or a firewall is interfering with things.  Sometimes.  You should search this forum for other discussions covering the subject.  #14 isn't bad, although at almost 4 months old, it is from prehistoric times.  Still, the advice in there about what to do about antivirus & firewalls is still valid.
